src/jdk.compiler/share/classes/com/sun/tools/javac/parser/DocCommentParser.java

```diff
@@ -593,6 +593,7 @@ private List<DCTree> inlineContent() {
         int depth = 1;
         textStart = -1;
 
+        loop:
```

Comment:
This label doesn't seem to be used, why was it added back?

Reply:
It was just to try and minimize changes with downstream commits, such as PR #13431, and didn't seem important to clean up in this round.